crates/forge-runner/src/gas/utils.rs

```diff
@@ -0,0 +1,27 @@
+use starknet_types_core::felt::Felt;
+
+pub(super) fn shorten_felt(felt: Felt) -> String {
+    let padded = format!("{felt:#066x}");
+    let first = &padded[..4];
+    let last = &padded[padded.len() - 4..];
+    format!("{first}…{last}")
+}
```

Comment:
The first 3 characters area always `0x0`, maybe we should also show 4 after that? It's not very useful to show that beginning otherwise.